INCODE/AA1/sensehat/schema.json

```diff
@@ -0,0 +1,71 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "$schemaVersion": "0.1.0",
+    "modelTags": "INCODE",
+    "$id": "https://smart-data-models.github.io/dataModel.SenseHat/schema.json",
```

Comment:
I don't think that there will be a subject named SenseHat, anyhow this does not prevent the validation because it will be modified in the publication process

Reply:
I modified it this way: 
    "$id": "https://smart-data-models.github.io/dataModel.Device/SenseHat/schema.json",